content/en/docs/reference/config/analysis/ist0113/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: MTLSPolicyConflict
+layout: analysis-message
+---
+
+This message occurs when a destination rule resource and a policy resource are
+in conflict with regards to mutual TLS. The two resources are in conflict if they each
+specify incompatible mutual TLS modes to be used; this conflict means traffic matching
+the destination rule to the specified host will be rejected.
+
+This message is only produced on clusters that are not using [automatic mutual
+TLS](/docs/tasks/security/authentication/auto-mtls/).
+
+## An example
+
+Consider an Istio mesh with the following mesh policy:
+
+{{< text yaml >}}
+apiVersion: authentication.istio.io/v1alpha1
+kind: MeshPolicy
+metadata:
+  name: default
+spec:
+  peers:
+  - mtls: {}
+{{< /text >}}
+
+The effect of this policy resource is that all services have an authentication
+policy requiring mutual TLS to be used. However, note that without a corresponding
+destination rule requiring traffic to use mutual TLS, traffic will be sent to services
+without using mutual TLS. This conflict means that traffic destined for services in
+the mesh will ultimately fail.
+
+In this example, you can fix the issue in one of two ways: you could downgrade
+the mesh policy's mutual TLS requirements to accept plaintext traffic (which might include
+removing the mesh policy entirely), or you can create a corresponding
+destination rule that specifies to use mutual TLS for traffic within the mesh:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: mtls-for-cluster
+spec:
+  host: *.svc.cluster.local
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+{{< /text >}}
+
+## Which destination rules and policies are relevant to a service
+
+To effectively resolve mutual TLS conflicts, it's helpful to understand how both
+destination rules and policies affect traffic to a service. Consider an example
+service named `my-service` in namespace `my-namespace`. To determine which
+policy object is applied to `my-service`, the following resources are matched in
+order:
+
+1. A policy resource in namespace `my-namespace` that contains a `target`
+   specifying `my-service`.
+1. A policy resource in namespace `my-namespace` that does not contain a
+   `target`. This policy implicitly applies to the entire namespace.
```

Comment:
this ns scope is intended, not sure why we call it implicit?

Reply:
I don't mean that its unintended by saying that its implicit. I mean it's implicit because the namespace itself is not listed as a target the same way that a service does (by listing no targets and naming it default, the target is the namespace even though it's not explicitly written).